browser/src/control/Control.Zotero.js

```diff
@@ -785,6 +785,14 @@ L.Control.Zotero = L.Control.extend({
 		app.socket.sendMessage('commandvalues command=.uno:TextFormFields?type=vnd.oasis.opendocument.field.UNHANDLED&commandPrefix=ADDIN%20ZOTERO_ITEM%20CSL_CITATION');
 	},
 
+	refreshCitations: function() {
+		//discard the cached url and fetch fresh one
+		var refreshURL = 'https://api.zotero.org/users/' + this.userID + '/items/top' + this.getZoteroItemQuery() + '&itemKey=' + this.getCitationKeys().join(',');
+		if (this._cachedURL[refreshURL])
+			delete this._cachedURL[refreshURL];
+		this.updateCitations();
```

Comment:
I don't see updateCitations function here

Reply:
This PR depends on #5636